lib/addons/external-dns/index.ts

```diff
@@ -22,9 +22,9 @@ const defaultProps = {
     name: 'external-dns',
     chart: 'external-dns',
     namespace: 'external-dns',
-    repository: 'https://charts.bitnami.com/bitnami',
+    repository: 'https://kubernetes-sigs.github.io/external-dns/',
     release: 'blueprints-addon-external-dns',
-    version: '6.20.3',
+    version: '1.12.0',
```

Comment:
fyi the latest version is at 1.13. Not sure if that one's ARM compatible.

Reply:
ya nice catch, fixed. It works now with `1.13.0`